github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS

```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - phoracek
```

Comment:
Please add `qinqon` here and to approvers.

BTW isn't this dangerous? Let's say I change my nick on GitHub and someone else takes over. Do they get approver rights everywhere?

Reply:
This config gives you permission only for the `cluster-network-addons-operator/` directory. About the username change, you might be right.

Reply:
As far as I remember, you need to be at least a contributer to a repo or org

Reply:
Thanks